src/historywork/WriteVerifiedCheckpointHashesWork.cpp

```diff
@@ -14,36 +15,74 @@
 #include "work/ConditionalWork.h"
 #include <Tracy.hpp>
 #include <algorithm>
+#include <filesystem>
 #include <fmt/format.h>
 
 namespace stellar
 {
+std::optional<LedgerNumHashPair>
+WriteVerifiedCheckpointHashesWork::loadLatestHashPairFromJsonOutput(
+    std::filesystem::path const& path)
+{
+    if (!std::filesystem::exists(path))
+    {
+        throw std::runtime_error("file not found: " + path.string());
```

Comment:
I don't love throwing a runtime error here, but its what we do in `loadHashFromJsonOutput` as well so maybe I should keep it? Could just log errors / return nullopt and exit more gracefiully at a higher level.